site/public/css/table.css

```diff
@@ -108,6 +116,11 @@
     .mobile-table th:last-child {
         padding-right: 15px;
     }
+    
+    .mobile-table tr:nth-child(even) td {
+        background: var(--standard-light-gray);
+    }
+    
```

Comment:
Instead of making this change, what about using the existing `.table-striped` class on the relevant table?